applications/FluidDynamicsApplication/custom_constitutive/newtonian_3d_law.cpp

```diff
@@ -106,6 +104,11 @@ double Newtonian3DLaw::GetEffectiveViscosity(ConstitutiveLaw::Parameters& rParam
     return rParameters.GetConstitutiveMatrix()(5,5);
 }
 
+double Newtonian3DLaw::ComputeEffectiveViscosity(ConstitutiveLaw::Parameters& rParameters) const {
```

Comment:
I think that virtual specifier is missing...

Reply:
it is in the header. I think it should only be there, right?